server/schemas/index.js

```diff
@@ -0,0 +1,38 @@
+import { GraphQLSchema, GraphQLObjectType, GraphQLList, GraphQLString } from 'graphql';
```

Comment:
Perhaps, this first line should be like you did the `server/schemas/types/expense.type.js` first line; step down each imported item for clarity.

Reply:
👍